onyo/utils.py

```diff
@@ -68,10 +68,10 @@ def get_git_root(path):
 
 def get_full_filepath(git_directory, file):
     full_filepath = os.path.join(git_directory, file)
-    if not os.path.isfile(full_filepath):
+    if not os.path.exists(full_filepath):
```

Comment:
Unsure what `exists()` checks for, but in shell it is very broad and includes symlinks, sockets, etc.

I'm not sure if Onyo should care, and if someone finds a way to abuse features for their use-case, more power to them. But I just wanted to register that this might be broader than just files/folders.

Reply:
While this is not the last word on the topic, my thought was to allow these and all other types of files, as long as they "exist". This might introduce weird things and behaviour, but as long as I don`t see anything breaking for sure (which seems not to be the case when building a path) I prefered to allow instead of block stuff with onyo.

As a note: when it definitely does matter (like for mkdir, mv, etc) the onyo commands check more specificly, what the path contains.

Reply:
Sounds good to me. :-)